src/tests/integration/test_clickhouse.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+from matplotlib import pyplot as plt
+
+
+def test_query_count(ip_with_clickhouse, test_table_name_dict):
```

Comment:
don't we have parametrized tests already (for all the tests in this file)? so we don't have to repeat the code and just add clickhouse?

Reply:
I'm not sure I understand.
I think those tests are in test_generic_db_operations? There are separate DB test files also.

Reply:
so all these tests look pretty generic; none of them suggest that they're using clickhouse-only features. so I think it'd be more maintainable to parametrize them instead of repeating them.

I took a quick look and it seems like we're indeed duplicating these tests: https://github.com/search?q=repo%3Aploomber%2Fjupysql%20test_query_count&type=code

since this isn't part of the original AC, I'll open a new issue.